pi_controller/cattown/.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
```

Comment:
The actual OS run on the Pi may end up being [Raspberry Pi OS Lite 64-bit](https://www.raspberrypi.com/software/operating-systems/).  I'm not sure if it matters whether we run tests on that or ubuntu-latest though?

Reply:
Ooo yea good question. This is honestly template code that I didn't modify at all... I haven't yet even figured out how to get it hooked up to GitHub Actions. So it's not currently running. I think I'll ask to "won't fix" this template code for the moment, and then we'll want to figure out whether GitHub Actions can do a release build on Raspberry Pi or not. 

Note that it's still an open question how we'll do release management / deployment to edge devices — something we'll need to think about soon if we're going to start shipping beta units. My impulse would be to use the OSS version of Mender since it's free... but there's some work involved in hosting and setting that up, and I'm not sure whether there's a better option out there for small teams like us.